components/footer/footer-group.tsx

```diff
@@ -0,0 +1,69 @@
+import React from 'react'
+import useTheme from '../use-theme'
+import { useFooterContext } from './footer-context'
+
+interface Props {
+  title: string | React.ReactNode
+}
+
+const defaultProps = {
+  title: '',
+}
+
+type NativeAttrs = Omit<React.HTMLAttributes<any>, keyof Props>
+export type FooterGroupProps = Props & typeof defaultProps & NativeAttrs
+
+const FooterGroup: React.FC<React.PropsWithChildren<FooterGroupProps>> = ({
+  title,
+  children,
+  ...props
+}) => {
+  const theme = useTheme()
+  const { breakPoint } = useFooterContext()
+  const [expand, setExpand] = React.useState(false)
+  return (
+    <div {...props}>
+      <h3 onClick={() => setExpand(!expand)}>{title}</h3>
```

Comment:
The `title` should be compatible with both `string` and `React.Elements`, (or React Component?)
otherwise we can't customize the font and size and other styles.

Reply:
Here, if `title` is an element, then h3 is redundant.

Reply:
Of course, after replacing it with an element, there are still events to consider.

Reply:
Fair Point. On it ✅